test-apps/test-tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,36 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs): m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs) {
+        m_servoPwmOut.period_ms(20);
+}
+
+float TutorialServo::getServoRangeInDegrees( ) const{
+    return m_servoRangeInDegrees;
+}
+
+float TutorialServo::getMinPulseWidthInMs( ) const{
+    return m_minPulsewidthInMs;
+}
+
+float TutorialServo::getMaxPulseWidthInMs( ) const{
+    return m_maxPulsewidthInMs;
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees){
+    if (degrees < 0){
+        printf("Out of Range. Setting to min");
```

Comment:
printfs are awesome for debugging, but for the final driver implementation it's best to take them out. If you want to output info messages, I suggest putting the printfs in main.cpp.